hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/WorldLedgers.java

```diff
@@ -146,6 +151,19 @@ public boolean defaultKycStatus(final TokenID tokenId) {
                 tokenId, ACC_KYC_GRANTED_BY_DEFAULT, StaticEntityAccess::defaultKycStatus);
     }
 
+    public Optional<EvmTokenInfo> evmInfoForToken(
```

Comment:
Do we plan to remove the `Optional<TokenInfo> infoForToken()` method below? Or do we still need it until something more is added to `EvmTokenInfo`?